xbmc/win32/Win32DelayedDllLoad.cpp

```diff
@@ -70,5 +70,5 @@ FARPROC WINAPI delayHookFailureFunc (unsigned dliNotify, PDelayLoadInfo pdli)
 }
 
 // assign hook functions
-PfnDliHook __pfnDliNotifyHook2 = delayHookNotifyFunc;
-PfnDliHook __pfnDliFailureHook2 = delayHookFailureFunc;
```

Comment:
Not sure yet why it breaks but update 3 is pre-release so it could be a bug. Currently installing it to investigate as it might be something we need fixed before it goes rtm.